.idea/KI-Client.iml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module classpath="CMake" type="CPP_MODULE" version="4">
+  <component name="sonarModuleSettings">
```

Comment:
Hast du sonar in clion integriert? musst mir mal sagen wie das geht, das brauch ich auch :D

Reply:
Das war aus LibCommon übernommen.
Aber das sollte das [SonarQube-Plugin](https://github.com/sonar-intellij-plugin/sonar-intellij-plugin) sein, das man sich über Settings holen kann.

Reply:
Funktioniert das bei dir? Ich bekomme das nicht so ganz zum laufen, und das Feature mit lokaler analyse scheint deprecated zu sein...